### PR TITLE
Add confirmation of funds to developer portal

### PR DESCRIPTION
**Pull request by isabellaio**
# Summary

_(What is this feature about)_

# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Documentation is updated
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: docs/de/developer-portal-page.md [hunk 1]
```diff
@@ -225,6 +225,38 @@ Um den Transaction Status eines Payments abzufragen, fügen Sie Ihre Payment-Id
...
+Der nachfolgende Code beschreibt einen beispielhaften cURL Command, der Confirmation of Funds für PSU "PSU-Successful" durchführt:
+
+`POST https://sandbox-api.dev.adorsys.de/v1/funds-confirmations`
+
+```sh
+curl -v "https://sandbox-api.dev.adorsys.de/v1/payments/sepa-credit-transfers" \
+  -H "accept: application/json" \
+  -H "X-Request-ID: 99391c7e-ad88-49ec-a2ad-99ddcb1f7721" \
+  -H "Content-Type: application/json" \
+  -H "tpp-redirect-uri: https://adorsys.de/" \
```
Comment (atamanroman) [inline docs/de/developer-portal-page.md]: This header should not be required.

File: docs/en/developer-portal-page.md [hunk 2]
```diff
@@ -271,6 +271,36 @@ Payment Status Endpoint.
...
+The following code snippet is an example cURL command which performs confirmation of funds for PSU "PSU-Successful":
+
+`POST https://sandbox-api.dev.adorsys.de/v1/funds-confirmations`
+
+```sh
+curl -v "https://sandbox-api.dev.adorsys.de/v1/payments/sepa-credit-transfers" \
+  -H "accept: application/json" \
+  -H "X-Request-ID: 99391c7e-ad88-49ec-a2ad-99ddcb1f7721" \
+  -H "Content-Type: application/json" \
+  -H "tpp-redirect-uri: https://adorsys.de/" \
```
Comment (atamanroman) [inline docs/en/developer-portal-page.md]: This header should not be required.

File: docs/en/developer-portal-page.md [hunk 3]
```diff
@@ -271,6 +271,36 @@ Payment Status Endpoint.
...
+        "instructedAmount": {
+          "currency": "EUR",
+          "amount": "15"
+        }
+      }'
+```
+
 ### Dedicated Consent Creation
 
 In order to create a dedicated consent, replace the Iban in your request with the
```
Comment (atamanroman) [inline docs/en/developer-portal-page.md:305]: Please delete the log file below (can't comment there) and add `service/logs` to our `.gitignore`. It's from a bug in our logging config (testing only).

**Review by atamanroman — CHANGES_REQUESTED** [review]

File: docs/de/developer-portal-page.md (hunk 1 above)
Comment (isabellaio) [reply docs/de/developer-portal-page.md]: done

File: docs/en/developer-portal-page.md (hunk 2 above)
Comment (isabellaio) [reply docs/en/developer-portal-page.md]: done

File: docs/en/developer-portal-page.md (hunk 3 above)
Comment (isabellaio) [reply docs/en/developer-portal-page.md:305]: done

File: docs/de/developer-portal-page.md [hunk 4]
```diff
@@ -225,6 +225,37 @@ Um den Transaction Status eines Payments abzufragen, fügen Sie Ihre Payment-Id
 | :------------- | :--------------------- | :--------- | :----------------- |
 | PSU-Successful | DE11760365688833114935 | finalised  | ACTC/ACSP          |
 
+
+### Confirmation of Funds
+
+Confirmation of Funds überprüft, ob der zu überweisende Betrag auf einem Konto vorhanden ist. Wenn der Kontostand ausreichend ist, wird "true" ausgegeben, anderenfalls wird "false" ausgegeben.
```
Comment (wandi34) [inline docs/de/developer-portal-page.md:231]: The format of true and false is inconsistent to the english doc. Maybe we can use bold text? italic? or code format?

Comment (isabellaio) [reply docs/de/developer-portal-page.md:231]: done

**Review by atamanroman — APPROVED** [review]

**Pull request merged**